src/main/java/fr/xephi/authme/datasource/SQLite.java

```diff
@@ -601,12 +601,45 @@ private PlayerAuth buildAuthFromResultSet(ResultSet row) throws SQLException {
             .locX(row.getDouble(col.LASTLOC_X))
             .locY(row.getDouble(col.LASTLOC_Y))
             .locZ(row.getDouble(col.LASTLOC_Z))
-            .locWorld(row.getString(col.LASTLOC_WORLD));
+            .locWorld(row.getString(col.LASTLOC_WORLD))
+            .locYaw(row.getFloat(col.LASTLOC_YAW))
+            .locPitch(row.getFloat(col.LASTLOC_PITCH));
 
         String ip = row.getString(col.IP);
         if (!ip.isEmpty()) {
             authBuilder.ip(ip);
         }
         return authBuilder.build();
     }
+
+
+    private static void close(Statement st) {
```

Comment:
Thanks for the review. 
When I tested this locally the sqlite driver was implementing an older version of jdbc such that the `isClosed` method doesn't exist. So I added back the specific implementations. I kept the utils class, thinking Sqlite will be updated sometime, which will allow us to use the same methods. It's probably more sensible to revert it and to create a utils class when it makes sense, so I'll do that :)